packages/@webex/plugin-meetings/src/meeting-info/meeting-info-v2.ts

```diff
@@ -208,7 +208,7 @@ export default class MeetingInfoV2 {
       .request({uri: conversationUrl, qs: {includeParticipants: true}, disableTransform: true})
       .then(({body: conversation}) => {
         const body = {
-          title: conversation.displayName,
+          title: conversation.displayName ? conversation.displayName : 'AdHoc Meeting',
```

Comment:
Add/modify a test case to test this

Reply:
Can we pass the name from first participant from participant list(as per WWC) if  conversation.displayName is not available?

Reply:
The title should anyway come from **App.js**, right?

How does WWC do it today?

Reply:
@mkesavan13 I had keep a dummy tittle as a string.Genrally it is the conversation title for the 1:1 meeting there is no conversation title so i had keep the condition.I had a thought in my mind like to keep an empty string if the conversation title was not defined.

Reply:
As @rsarika suggestion can we try like that or we can keep .

Reply:
@rsarika yes i think so in the test cases they had mentioned only the group space titles not the 1:1 meeting titles. we need to look through this

Reply:
If this is a bug in SDK that we are fixing, how does Webex Web Client handle it?

They should have hit this issue as well, right? Did we check how does WWC handle this one?

Reply:
@mkesavan13, At present no,I had seen the webex web client but i could not see the Conversation title or Space Instant API s request body to create an AdHoc meeting like that.

Reply:
@mkesavan13 In the app.js i did not encounter error and it has created meeting object but due to conversation title missing in the request body the create meeting method could not create the meeting info.Ideally if we give the title for the create AdHoc meeting it will give the response meeting info that was used in the method return.